examples/vm_scheduling/reinforcement_learning/ac/agent/ac_net.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+from torch.distributions import Categorical
+
+from maro.rl import DiscreteACNet, OptimOption
+
+
+class ACNet(DiscreteACNet):
+    def forward(self, states, actor: bool = True, critic: bool = True):
+        states = torch.from_numpy(np.asarray(states)).to(self.device)
+        if len(states.shape) == 1:
+            states = states.unsqueeze(dim=0)
+        return (
+            self.component["actor"](states) if actor else None,
+            self.component["critic"](states) if critic else None
+        )
+
+    def get_action(self, states, legal_action, training=True):
```

Comment:
Can you make legal_action a part of ``states`` so we don't have to change the call interface? There is no restriction on what type ``states`` should be in ``forward``.

Reply:
use a tuple (states, legal_action) as states to input the function